app/src/main/java/framgia/com/ichat/screen/home/HomeActivity.java

```diff
@@ -55,4 +71,13 @@ public boolean onQueryTextSubmit(String s) {
     public boolean onQueryTextChange(String s) {
         return false;
     }
+
+    @Override
+    public void setImageAvatar(String url) {
+        if (url!=null){
+            GlideApp.with(this).load(url).circleCrop().into(mImageAvartar);
+            return;
+        }
+        GlideApp.with(this).load(Constants.UserProfile.DEFAULT_PROFILE_URL).circleCrop().into(mImageAvartar);
```

Comment:
em có thể sử lý trường hợp url == null với method 
```suggestion
        GlideApp.with(this)
                                      .load(url)
                                      .error(Constants.UserProfile.DEFAULT_PROFILE_URL)
                                      .circleCrop()
                                      .into(mImageAvartar);
```